_includes/left-nav.html

```diff
@@ -148,6 +148,7 @@ <h3 class="{% if '/faq/index.html' == page.url %} active {% endif %}"> <a href="
     <h3 class="{% if '/faq/index.html' == page.url %} active {% endif %}"> <a href="{{ site.baseurl }}/help/faq">Frequently Asked Questions </a></h3>
     <h3 class="{% if '/data_faq/index.html' == page.url %} active {% endif %}"> <a href="{{ site.baseurl }}/help/data_faq">Data and Curation FAQs </a></h3>
     <h3 class="{% if '/api/index.html' == page.url %} active {% endif %}"> <a href="{{ site.baseurl }}/help/api">Using the ADS API </a></h3>
+	<h3 class="{% if '/legacy/index.html' == page.url %} active {% endif %}"> <a href="{{ site.baseurl }}/help/terms">Legacy Services</a></h3>
```

Comment:
Do you want this to be one big long page (like the existing What's New or FAQ pages) or a drop down menu with subpages (like Quick Start or Making a Query)? The way you have it now will make one long page, though there's a typo - `site.baseurl }}/help/terms` should be `site.baseurl }}/help/legacy`. If you prefer a dropdown, copy and paste one of the div sections just above and change everything to point to `legacy`

Reply:
One big, long page is fine. Thanks for catching that typo! :)